go/teams/list.go

```diff
@@ -469,85 +471,53 @@ func AnnotateSeitanInvite(ctx context.Context, team *Team, invite keybase1.TeamI
 	switch labelType {
 	case keybase1.SeitanKeyLabelType_SMS:
 		sms := label.Sms()
-		var smsName string
+		smsName := blankSeitanLabel
 		if sms.F != "" && sms.N != "" {
 			smsName = fmt.Sprintf("%s (%s)", sms.F, sms.N)
 		} else if sms.F != "" {
 			smsName = sms.F
 		} else if sms.N != "" {
 			smsName = sms.N
 		}
-		return keybase1.TeamInviteName(smsName), nil
+		return keybase1.TeamInviteDisplayName(smsName), nil
+	case keybase1.SeitanKeyLabelType_GENERIC:
+		return keybase1.TeamInviteDisplayName(label.Generic().L), nil
 	default:
-		return "", nil
+		return keybase1.TeamInviteDisplayName(blankSeitanLabel), nil
 	}
 }
 
-type AnnotatedInviteMap map[keybase1.TeamInviteID]keybase1.AnnotatedTeamInvite
+func AnnotateInvitelink(mctx libkb.MetaContext, team *Team, invite keybase1.TeamInvite) (name keybase1.TeamInviteDisplayName, err error) {
+	pkey, err := SeitanDecodePKey(string(invite.Name))
+	if err != nil {
+		return name, err
+	}
+	keyAndLabel, err := pkey.DecryptKeyAndLabel(mctx.Ctx(), team)
+	if err != nil {
+		return name, err
+	}
 
-func AnnotateInvites(ctx context.Context, g *libkb.GlobalContext, team *Team) (AnnotatedInviteMap, error) {
```

Comment:
I deleted this function since it was overkill for the one place that was using it.